src/nextcloud/talk/recording/Participant.py

```diff
@@ -208,7 +209,7 @@ def __del__(self):
             # created in "/tmp".
             self.driver.quit()
 
-    def startChrome(self, width, height, env):
+    def startChrome(self, width, height, env, executable_path):
```

Comment:
Although in Python _snake_case_ is typically used (which kind of makes sense :-P ) this project uses _lowerCamelCase_ for variable/parameter/attribute names instead.

Besides that it should be called something like _driverPath_ rather than _executablePath_, as _executable_ is too broad and could also mean the browser executable instead of the driver.

Reply:
I will take this into account, but I think I chose this name because this is the name Selenium uses.

Reply:
Ah, I see :-) Nevertheless, the context of Selenium and the `Participant` object is different, so that is why I preferred it to be called `driverPath` even if, internally, it is passed to Selenium as `executable_path`.